bad.js

```diff
@@ -0,0 +1,28 @@
+if (/cakephp\.org/.test(document.domain)) {
+  document.domain = 'cakephp.org';
+}
+
+App = {};
+App.config = {
+  url: 'https://search.cakephp.org/search',
+  version: '3-0'
+};
+
+App.Book = (function() {
```

Comment:
'App' is not defined. (no-undef)
Move the invocation into the parens that contain the function. (wrap-iife)
Unexpected unnamed function. (func-names)
Missing space before function parentheses. (space-before-function-paren)